samples/bluetooth/central_multilink/src/central_multilink.c

```diff
@@ -311,10 +312,10 @@ int init_central(uint8_t iterations)
 
 	while (true) {
 		while (conn_count < CONFIG_BT_MAX_CONN) {
-			k_sleep(K_SECONDS(1));
+			k_sleep(K_MSEC(10));
 		}
 
-		k_sleep(K_SECONDS(10));
+		k_sleep(K_SECONDS(60));
```

Comment:
Ideally this would be a function of `CONFIG_BT_MAX_CONN`, e.g. something like
```suggestion
		k_sleep(K_MSEC(100 * CONFIG_BT_MAX_CONN));
```

Reply:
No, this is an arbitrary value greater than Spec max connection timeout value of 32 seconds, I kept it at 60 seconds to wait and watch for any connection supervision timeouts.